docs/modernization/WebViewCompatible.md

```diff
@@ -0,0 +1,224 @@
+---
```

Comment:
@normesta / @mcleanbyron Could you review this new documentation for Web View Compatible? This new control augments the Win32 web view provided previously by permitting fallback to Trident. We'll need to update WebView documentation ❔ to introduce the concept